src/icon.scss

```diff
@@ -13,6 +13,12 @@ $blockTNT: sap-icon-TNT;
 
   font-style: normal;
   display: inline-block;
+
+  @include fd-focus() {
```

Comment:
same

Reply:
Hi Yuri,

It uses _outline-offset_ which is not supported in IE11, to avoid this issue used fd-focus.